OutOfSchool/OutOfSchool.DataAccess/OutOfSchool.DataAccess.csproj

```diff
@@ -28,12 +28,12 @@
       <PackageReference Include="H3Lib" Version="3.7.2" />
       <PackageReference Include="Microsoft.AspNetCore.DataProtection.EntityFrameworkCore" Version="6.0.6" />
       <PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="6.0.6" />
-      <PackageReference Include="Microsoft.EntityFrameworkCore" Version="6.0.6" />
-      <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="6.0.6">
+      <PackageReference Include="Microsoft.EntityFrameworkCore" Version="7.0.2" />
+      <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="7.0.2">
```

Comment:
(Same version for all references) Shouldn't specific version be set using solution-level `Directory.Packages.props`?